pkg/sql/copy.go

```diff
@@ -244,7 +245,9 @@ func (c *copyMachine) run(ctx context.Context) error {
 	case tree.CopyFormatText:
 		c.textDelim = []byte{c.delimiter}
 	case tree.CopyFormatCSV:
-		c.csvReader = csv.NewReader(&c.buf)
+		c.csvInput.Reset()
+		c.csvReader = csv.NewReader(&c.csvInput)
+		//c.csvReader = csv.NewReader(&c.buf)
```

Comment:
hehe, you put the delete on the wrong commit
oh well, no biggie!

Reply:
whooooops